src/pages/Signup.js

```diff
@@ -0,0 +1,170 @@
+                                fullWidth
+                                id="email"
+                                label="Email Address"
+                                name="email"
+                                autoComplete="email"
+                                autoFocus
+                            />
+                            <TextField
+                                margin="normal"
+                                required
+                                fullWidth
+                                name="password"
+                                label="Password"
+                                type="password"
+                                id="password"
+                            />
+                            <TextField
+                                margin="normal"
+                                required
+                                fullWidth
+                                name="rePassword"
+                                label="Re-enter Password"
+                                type="password"
+                                id="rePassword"
+                            />
+                            <Grid container spacing={2}>
+                                <Grid container item xs={6} direction="column" >
+                                    <TextField
+                                        margin="normal"
+                                        required
+                                        fullWidth
+                                        type="tel"
+                                        pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}"
+                                        id="phoneNumber"
+                                        label="Phone Number"
+                                        name="phoneNumber"
+                                    />
+                                </Grid>
+                                <Grid container item xs={6} direction="column" >
+                                    <LocalizationProvider dateAdapter={AdapterDayjs}>
+                                        <DatePicker
+                                            label="Date of Birth"
+                                            margin="normal"
+                                            required
+                                            fullWidth
+                                            value={dob}
+                                            onChange={setDob}
+                                            renderInput={(params) => <TextField {...params} />}
+                                        />
+                                    </LocalizationProvider>
+                                </Grid>
+                            </Grid>
+                            <FormControlLabel
+                                control={<Checkbox value="accept" color="primary" />}
+                                label="I have read and accept the terms and conditions"
+                            />
```

Comment:
The same situation here for the button you're using. We will use the same layout for every button in the web for consistency